aten/src/ATen/native/PointwiseOps.cpp

```diff
@@ -41,6 +41,7 @@ TORCH_META_FUNC(addcdiv)
 }
 
 } // namespace meta
+
```

Comment:
i did it on purpose since there's usually a newline when a new namespace is introduced, unless it's after another namespace. so i'm in favor of keeping this